### ISSUE TITLE
Custom range does not work for new fires layers

### ISSUE THREAD

**Issue by richardbarad** [wri/gfw-mapbuilder#412]
If i:

1) Turn on the new fires layers
2) Select custom range
3) Change the start/end date to a custom range

I do not see any new calls being made in the console after changing the date. FYI, looks like the same issue is pres
[Fires_bug.zip](https://github.com/wri/gfw-mapbuilder/files/3339992/Fires_bug.zip)
ent in GFW Fires!

The fires continue to show data for the past 24 hours, instead instead of showing the data for the custom range. I am attaching video of the workflow.

**Comment by SwampGuzzler** [comment]
@richardbarad  Good catch - we were missing a proper layerDefinition on these `archive` layers.

You can test this [here](http://alpha.blueraster.io/gfw-mapbuilder/412-fires-custom-ranges/) !

**Comment by SwampGuzzler** [comment]
@SwampGuzzler See [this](https://github.com/wri/gfw-mapbuilder/pull/417) PR

**Comment by richardbarad** [comment]
@SwampGuzzler. Looks good! I noticed that the report in this build does not show the correct analysis module results (it still shows highcharts), but hopefully that will be resolved once merged into development?

Richard

**Comment by SwampGuzzler** [comment]
Sorry - this build was just a hotfix to show the Fires Layers updates -- when we merge this in and think about pushing anywhere all of the modules we be resolved properly

**Comment by richardbarad** [comment]
@SwampGuzzler. Thanks! From my side, this looks good to merge! Feel free to do a code review before merging. 

Richard
